torch/csrc/api/src/nn/modules/embedding.cpp

```diff
@@ -11,27 +11,79 @@
 namespace torch {
 namespace nn {
 
-EmbeddingOptions::EmbeddingOptions(int64_t count, int64_t dimension)
-    : count_(count), dimension_(dimension) {}
+EmbeddingOptions::EmbeddingOptions(int64_t num_embeddings, int64_t embedding_dim) : num_embeddings(num_embeddings), embedding_dim(embedding_dim) {}
 
-EmbeddingImpl::EmbeddingImpl(EmbeddingOptions options) : options(options) {
-  reset();
+EmbeddingImpl::EmbeddingImpl(int64_t num_embeddings, int64_t embedding_dim, c10::optional<int64_t> padding_idx, c10::optional<float> max_norm,
+float norm_type, bool scale_grad_by_freq, bool sparse, c10::optional<torch::Tensor> weight): EmbeddingImpl(EmbeddingOptions(num_embeddings, embedding_dim)){
```

Comment:
We should implement `EmbeddingImpl::EmbeddingImpl(EmbeddingOptions options)` here instead